app/src/main/java/com/yagni/view/HomeGUI.java

```diff
@@ -1,9 +1,8 @@
 package com.yagni.view;
-
 import java.awt.BorderLayout;
```

Comment:
double check your imports in your 3 classes and condense them by using:

import javax.swing.*;
import java.awt.*;
import java.awt.event.*;

etc.

Reply:
Changed now as well